src/keycloak/keycloak_openid.py

```diff
@@ -452,3 +457,63 @@ def get_permissions(self, token, method_token_info="introspect", **kwargs):
                     permissions += policy.permissions
 
         return list(set(permissions))
+
+    def uma_permissions(self, token, permissions, audience=SAME_AS_CLIENT, response_mode="permissions"):
+        """
+        The token endpoint is used to retrieve UMA permissions from Keycloak. It can only be
+        invoked by confidential clients.
+
+        http://openid.net/specs/openid-connect-core-1_0.html#TokenEndpoint
+
+
+        :param token: user token
+        :param permissions:
+        :return: permissions list
+        """
+
+        if audience is SAME_AS_CLIENT:
+            audience = self.client_id
+
+        permission = build_permission_param(permissions)
+
+        params_path = {"realm-name": self.realm_name}
+        payload = {"grant_type": "urn:ietf:params:oauth:grant-type:uma-ticket", "permission": permission,
+                   "response_mode": response_mode, "audience": audience, "Authorization": "Bearer "+token}
```

Comment:
I do not believe that the Authorization is needed in the payload based on the Keycloak docs https://www.keycloak.org/docs/latest/authorization_services/index.html#_service_obtaining_permissions

Reply:
agreed, removed.